### ISSUE TITLE
True Label Question

### ISSUE THREAD

**Issue by laowangzi** [yusanshi/NewsRecommendation#43]
Hi,
   Excellent work. However, I have a question. 
   In the train section of train.py, your code is` y = torch.zeros(len(y_pred)).long().to(device)`. The true label I think is `1 + 0*K`, rather than `0 * (K + 1)`.
